linux/jdk/debian/src/main/packaging/microsoft/17/debian/compat

```diff
@@ -0,0 +1 @@
+11
```

Comment:
I assume this is a debian compatibility value and therefore 11 makes sense

Reply:
yup it is